nestkernel/conn_builder.cpp

```diff
@@ -85,11 +85,13 @@ nest::ConnBuilder::ConnBuilder( Network& net,
   // be possible to set the delay without the weight.
   default_weight_ = !syn_spec->known( names::weight );
 
+  default_delay_ = !syn_spec->known( names::delay );
```

Comment:
@naveau My mistake, the `default_*` are members because they are also needed in `single_connect_()`; so they cannot be const.